hummingbot/connector/exchange/bittrex/bittrex_auth.py

```diff
@@ -59,8 +56,11 @@ def construct_content_hash(body: Dict[str, any] = {}) -> Tuple[str, bytes]:
             "Content-Type": "application/json",
             "Accept": "application/json",
         }
+        return headers
 
-        if subaccount_id:
-            headers.update({"Api-Subaccount-Id": subaccount_id})
-
-        return {"headers": headers, "body": content_bytes, "url": url}
+    def generate_WS_auth_params(self):
+        timestamp = self.time_provider.time()
```

Comment:
I realise that the time() here is in seconds. Is this suppose to be the case since I notice for REST authentication, it is in milliseconds

Reply:
Updated the timestamp to correct format